design-at-cornell/src/clubs/Clubs.tsx

```diff
@@ -5,30 +5,38 @@ import Pagination from '../pagination/Pagination';
 import { VerticalFlex } from '../components/ContainerStyles';
 import { Club } from '../../../server/src/types';
 import Dashboard from './dashboard/Dashboard';
+import { mobileBreakpoint } from '../constants/styling';
 
 const Clubs = () => {
   useEffect(() => {
     api
       .get('/getClubs')
       .then((res) => res.data.data)
       .then(setClubs);
+    changeClubsPerPage();
   }, []);
 
+  const changeClubsPerPage = () => {
+    if (window.innerWidth < mobileBreakpoint) {
+      setMobileVersion(8);
+    }
+  }
+
   const [clubs, setClubs] = useState<Club[]>([]);
   const [currentPage, setPage] = useState(1);
+  const [mobileVersion, setMobileVersion] = useState(20)
```

Comment:
Not sure if `mobileVersion` is a good name for this state, as I would think it be a boolean state rather than a number. Perhaps `cardsPerPage`,`cardsToDisplay`, or something like that.

Reply:
i changed the variable names in the previous commit above.